src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerSession.cs

```diff
@@ -161,20 +163,20 @@ internal sealed class PythonAnalyzerSession {
                 return;
             }
 
-            double privateMB;
-            double peakPagedMB;
+            double privateMb;
```

Comment:
`MB` is a unit, hence why I didn't do `Mb`.

Reply:
+1

Reply:
Sorry, autoformatting :(